app/lib/forms/choose_your_provider.rb

```diff
@@ -24,9 +24,9 @@ def previous_step
         :possible_funding
       elsif course.ehco?
         if eligible_for_funding?
-          :aso_possible_funding
+          "ehco_possible_funding"
         else
-          :funding_your_aso
+          "funding_your_ehco"
```

Comment:
Strings? Not an issue of course but it's not consistent with other returned step names.

Reply:
yeah i cant remember why symbols stopped working here maybe need to re-visit